lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/context/EvaluatorContext.java

```diff
@@ -170,6 +170,26 @@ public synchronized void submitContext(final Configuration contextConfiguration)
     this.contextControlHandler.send(contextControlProto);
   }
 
+  public synchronized void submitContext(final String contextConf) {
```

Comment:
May be another way around. Let submitContext(final Configuration contextConfiguration) to call this new one with deserialized string?